src/integrationTest/java/org/opensearch/security/privileges/PrivilegesEvaluatorDNFOFTest.java

```diff
@@ -0,0 +1,51 @@
+package org.opensearch.security.privileges;
+
+import com.carrotsearch.randomizedtesting.annotations.ThreadLeakScope;
+import org.apache.hc.core5.http.HttpStatus;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.opensearch.test.framework.TestSecurityConfig;
+import org.opensearch.test.framework.cluster.ClusterManager;
+import org.opensearch.test.framework.cluster.LocalCluster;
+import org.opensearch.test.framework.cluster.TestRestClient;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.equalTo;
+import static org.opensearch.test.framework.TestSecurityConfig.AuthcDomain.AUTHC_HTTPBASIC_INTERNAL;
+
+@RunWith(com.carrotsearch.randomizedtesting.RandomizedRunner.class)
+@ThreadLeakScope(ThreadLeakScope.Scope.NONE)
+public class PrivilegesEvaluatorDNFOFTest {
```

Comment:
I see another class, DoNotFailOnForbiddenTests, is there a reason we need a new test class?

Reply:
Moved to existing testing file in: [92d475b](https://github.com/opensearch-project/security/pull/3236/commits/92d475bf02c966ae6359eeb222f234cd6766a003). My only concern now is that it seems to be an integration test file, should I move the unit/regex matching tests to another file?